nautobot/ui/src/App.js

```diff
@@ -11,6 +11,9 @@ import { updateAuthStateWithSession, updateNavigation } from "@utils/store";
 import NautobotRouter from "./router";
 
 const theme = {
+    // TODO(norbert-mieczkowski-codilime): This should be fixed in Nautobot UI.
```

Comment:
Fixed in https://github.com/nautobot/nautobot-ui/pull/42 correct? Can we update to a new version of `nautobot-ui` to remove the need for this?

Reply:
I'll address this once #4538 is merged